Dockerfile

```diff
@@ -1,13 +1,14 @@
 FROM golang:1.5.1
 
-ADD ./cuberite_server /srv/cuberite_server
-ADD ./world /srv/world
-ADD ./docs/img/logo64x64.png /srv/logo.png
-ADD ./start.sh /srv/start.sh
-ADD ./go /go
-ADD ./docker_linux_x64/docker /bin/docker
+COPY ./cuberite_server /srv/cuberite_server
+COPY ./world /srv/world
+COPY ./docs/img/logo64x64.png /srv/logo.png
+COPY ./start.sh /srv/start.sh
+COPY ./go /go
+COPY ./docker_linux_x64 /bin
```

Comment:
Could we `ADD` the binaries from `get.docker.com` instead?
That would reduce the size of the build context + diffs would be much clearer when we change the versions we keep in the image.